pkgs/applications/altcoins/monero-gui/default.nix

```diff
@@ -105,7 +105,7 @@ stdenv.mkDerivation rec {
     description = "Private, secure, untraceable currency";
     homepage    = https://getmonero.org/;
     license     = licenses.bsd3;
-    platforms   = [ "x86_64-linux" ];
+    platforms   = filter (p: p != "x86_64-darwin") platforms.all;
```

Comment:
you can use meta.badPlatforms to do the same declaratively.

Reply:
Thank you, I didn't know about it.